index.js

```diff
@@ -176,6 +179,22 @@ class mochaPlugin {
               mocha.reporter(reporter, reporterOptions);
             }
 
+            const compilers = myModule.options['compilers'];
+            if (compilers !== undefined) {
+              const extensions = ['js'];
+              myModule.options['compilers'].split(' ').filter(function(e){return e!==''}).forEach(function (c) {
+                const idx = c.indexOf(':');
```

Comment:
Split magic! I didn't know regexp can be used in split.
Please tell me the reason for using reduce here? 🙂